### DISCUSSION TITLE
breakpoints for gallery not working on view.xml

### DISCUSSION THREAD

**Discussion by delfina-rossi** [breezefront/community#312] [1. Help] [answered]
Hello, I'm trying to change nav option for gallery only on mobile. (thumbs on desktop, dots on mobile).
On any standard theme it's possible by adding this on view.xml

```xml
<var name="breakpoints">
    <var name="mobile">
        <var name="conditions">
            <var name="max-width">991px</var>
        </var>
        <var name="options">
            <var name="options">
                <var name="nav">dots</var>
            </var>
        </var>
    </var>
</var>
```

but it's not working on breeze. Any help to get it to work?

**Comment by vovayatsyuk** [comment, marked as answer]
Breeze doesn't support `breakpoints` gallery feature, so you need to write styles for this.

Just [pushed the commit](https://github.com/breezefront/module-breeze/commit/c9d018db211cdfa57bd17f2588ed42ff58910ae0) that makes it easier to implement this feature for any media query.

Using from your theme:

```less
& when (@critical) {
    @media @media-md-down {
        .breeze-gallery:not(.opened) .thumbnails {
            .bz-gallery-nav-dots();
        }
    }
}

```

**Reply by delfina-rossi** [reply]
Thank you so much Vova, I'll give it a try.

**Comment by parthvisavadiya** [comment]
Go to vendor/magento/framework/View/Element/AbstractBlock.php and check which module do you get in this $module = $module ?: $this->getModuleName(); and set breakpoints in that module

Vendor/Module/etc/view.xml

<var name="breakpoints">
    <var name="mobile">
        <var name="conditions">
            <var name="max-width">991px</var>
        </var>
        <var name="options">
            <var name="options">
                <var name="nav">dots</var>
            </var>
        </var>
    </var>
</var>
